lib/bootstrap_form/helpers/bootstrap.rb

```diff
@@ -59,6 +59,15 @@ def static_control(*args, &block)
         end
       end
 
+      def custom_control(*args, &block)
+        options = args.extract_options!
+        name = args.first
+
+        form_group_builder(name, options) do
+          capture(&block)
+        end
+      end
```

Comment:
I see... So in order to support the example you described (`custom_control label: 'MyLabel'`), then we *do* need `extract_options!`. Right?

> `capture` is not needed since it is already done in `form_group`

Agreed. Can you update the PR to remove it?